### DISCUSSION TITLE
在hbuilder X 代码格式化 模板引擎 错位 你们如何解决的呀？

### DISCUSSION THREAD

**Discussion by sggcn** [layui/layui#1466] [问答]
我格式化后的效果 
![asd111](https://github.com/layui/layui/assets/131776613/d482c49b-a4f8-4461-98ae-b00e8ef921be)

**Comment by sentsim** [comment]
用 VSCode 或 Sublime Text 试试

**Comment by Sight-wcg** [comment]
安装 Prettier 插件，用 Prettier 格式化

**Reply by sggcn** [reply]
可以看看你的插件配置吗？我这边使用了插件没效果

**Reply by sggcn** [reply]
我是在*.html 下开发的

**Reply by Sight-wcg** [reply]
Prettier 配置文件中设置
embeddedLanguageFormatting:"off"

Hbuilder X 配置文件设置，注意是 `defaultFomat`, 不是 `defaultFormat`
 "defaultFomat.html_es6" : "formator-prettier"
